app/models/ability.rb

```diff
@@ -63,7 +65,8 @@ def case_worker(persona)
   end
 
   def provider_management
-    can %i[show index new create edit update change_password update_password find search], ExternalUser
+    can %i[index new create find search], ExternalUser
```

Comment:
As a refactor for naming and clarity consider renaming this method `can_administer_any_provider` and the method on line 103 as `can_administer_own_provider(persona)`

Reply:
👍